recipes/gdcm/all/conanfile.py

```diff
@@ -69,8 +81,12 @@ def generate(self):
         tc.variables["GDCM_BUILD_SHARED_LIBS"] = bool(self.options.shared)
         # FIXME: unvendor deps https://github.com/conan-io/conan-center-index/pull/5705#discussion_r647224146
         tc.variables["GDCM_USE_SYSTEM_EXPAT"] = True
+        tc.variables["GDCM_USE_SYSTEM_JSON"] = self.options.with_json
```

Comment:
I don't understand. What happen when with_json option is False? gdcm still uses json-c but bundle one?

Reply:
Moreover the json patch could be more robust, upstream doesn't rely on official config file.

Reply:
If I understood correctly, without GDCM_USE_SYSTEM_JSON json functionality just won't work. Same with openssl. Theoretically some people might want to use it but it's not part of core functionality - I didn't even know that gdcm has it for many years.